reference-implementation/to-upstream-wpts/writable-streams/bad-strategies.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+test(() => assert_throws(new TypeError(), () => new WritableStream({}, { size: 'a string' })),
```

Comment:
While it's impressive that you could fit this test on one line, I found it difficult to understand.

I'm sorry to be boring, but if you could split the test body into two statements I would appreciate it.